e2e/interchaintestv8/relayer_test.go

```diff
@@ -1532,3 +1746,208 @@ func (s *RelayerTestSuite) ConcurrentRecvPacketToCosmos(
 		s.Require().Equal(denomOnCosmos.IBCDenom(), resp.Balance.Denom)
 	}))
 }
+
+// Test_MigrateLightClientFromLatestReleaseVersion tests the migration of the light client from the latest released version
+// to the latest local binary version.
+func (s *RelayerTestSuite) Test_MigrateLightClientFromLatestReleaseVersion() {
```

Comment:
I think this level of migration tests are not good. It should be sufficient to do the migration tests in rust. You can import previous versions of `cw-ics08-wasm-eth` in Cargo.toml using git commit tag, and include it in your tests.

This would be my preferred way, not to use the compiled wasm binaries.

Reply:
Removed